server/controllers/medical/projects.js

```diff
@@ -20,15 +20,27 @@ exports.list = function list(req, res, next) {
   var sql;
 
   // send a larger response if complete is 1
-  if (req.query.complete === 1) {
+  if (req.query.complete === '1') {
     sql = 'SELECT project.id, project.enterprise_id, project.abbr, ' +
-      'project.zs_id, project.name ' +
+      'project.zs_id, project.name, project.locked ' +
     'FROM project;';
   } else {
     sql =
       'SELECT project.id, project.name FROM project;';
   }
 
+  if (req.query.unlocked === '1') {
+    sql = 'SELECT project.id, project.enterprise_id, project.abbr, ' +
+      'project.zs_id, project.name, project.locked ' +
+    'FROM project WHERE project.locked = \'0\' ;';
```

Comment:
For future reference, you do not need to write `project.locked = \'0\';` ... `projects.locked = 0;` should work fine.  It's much easier to read and edit.